handlers/handler-loadbalance/src/main/java/io/servicecomb/loadbalance/LoadbalanceHandler.java

```diff
@@ -350,4 +350,16 @@ private void loadFilter(String filter, LoadBalancer lb) {
       }
     }
   }
+
+  private boolean isNotChanged(String p, String s) {
```

Comment:
so complex?
it seems old logic is simpler.

Reply:
Yeah, the old logic is simpler. But after delete  configuration ,ServiceComb uses RoundRobin Rule as default ,then i  push configuration like RandomRule .ServiceComb still uses the  default configuration.

Reply:
yes, old logic is simpler but had some problems
but new logic is too confusion

i think eric-lee-ltk's suggestion is good.